### PR TITLE
go: added derived source support

### PR DESCRIPTION
**Pull request by hiteshbedre**

### CODE REVIEW

**Comment by ankit-privado** [comment]
Looking good to me, are you planning to add unit tests?

**Comment by hiteshbedre** [comment]
> Looking good to me, are you planning to add unit tests?

Test case will be part of upcoming PR.

**Review by ankit-privado — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ ThisBuild / version      := sys.env.getOrElse("BUILD_VERSION", "dev-SNAPSHOT")
 
 
 val cpgVersion        = "1.4.22"
-val joernVersion      = "2.0.92"
```
Comment (khemrajrathore) [inline build.sbt:11]: Don't upgrade to 98 we are seeing delta in comparison report

**Review by khemrajrathore — COMMENTED** [review]
@hiteshbedre Include test case as well, otherwise it is difficult to understand whether the sources we are looking at will be getting tagged or not

**Comment by hiteshbedre** [comment]
> @hiteshbedre Include test case as well, otherwise it is difficult to understand whether the sources we are looking at will be getting tagged or not

Added test cases here: d0220250af68fffb52af29ccbfdce5de190e6f0d
Covered cases:
1. Member Node tagging
2. Object having tagged member tagging.

File: build.sbt (hunk 1 above)
Comment (hiteshbedre) [reply build.sbt:11]: do you know which joern version is stable at current state? anyways we need to update to latest as Go's fixes are on the way.

Comment (khemrajrathore) [reply build.sbt:11]: we can upgrade till 2.0.92, I guess there is already a PR created by Datta on Dev

**Comment by pandurangpatil** [comment]
@khemrajrathore let the Joern upgrade done for this branch of `go`. We need to anyway get the diff issue resolved before the GO release before 6th Oct.

cc: @hiteshbedre

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**